ttnn/cpp/ttnn/operations/normalization/batch_norm/device/batch_norm_device_operation.cpp

```diff
@@ -34,6 +49,7 @@ void BatchNormOperation::validate_tensors(
 
     // weight (1, C, 1, 1)
     if (weight.has_value()) {
+        check_tensor_BN(weight.value());
         TT_FATAL(
             weight.value().get_logical_shape()[1] == C, "weight_shape[1] must be the same as input's channel size.");
```

Comment:
```suggestion
        check_tensor_BN(weight.value(), "weight_shape", C);
```
and please also remove the duplicate `TT_FATAL` below (the review tool won't let me include that line in my selection)